pkg/proxy/handlers/spacelister_list_test.go

```diff
@@ -22,127 +22,211 @@ import (
 	"github.com/codeready-toolchain/registration-service/pkg/proxy/metrics"
 	"github.com/codeready-toolchain/registration-service/pkg/signup"
 	"github.com/codeready-toolchain/registration-service/test/fake"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
 )
 
-func TestSpaceListerList(t *testing.T) {
-	fakeSignupService, fakeClient := buildSpaceListerFakes(t)
-
-	t.Run("HandleSpaceListRequest", func(t *testing.T) {
-		// given
-		tests := map[string]struct {
-			username             string
-			expectedWs           []toolchainv1alpha1.Workspace
-			expectedErr          string
-			expectedErrCode      int
-			expectedWorkspace    string
-			overrideSignupFunc   func(ctx *gin.Context, userID, username string, checkUserSignupComplete bool) (*signup.Signup, error)
-			overrideInformerFunc func() service.InformerService
-		}{
-			"dancelover lists spaces": {
-				username: "dance.lover",
-				expectedWs: []toolchainv1alpha1.Workspace{
+func TestSpaceListerListUserWorkspaces(t *testing.T) {
+	tests := map[string]struct {
+		username            string
+		expectedWs          func(*test.FakeClient) []toolchainv1alpha1.Workspace
+		expectedErr         string
```

Comment:
You don't really use `expectedErr` in these tests at all, do you?

Reply:
great catch, removed in 9327aafbac84aa39979ea90aea974f8954485941